config/environment.js

```diff
@@ -13,7 +13,7 @@ module.exports = function(environment) {
     appName                  : process.env.APP_NAME || 'Open Event',
     modulePrefix             : 'open-event-frontend',
     environment,
-    rootURL                  : '/',
+    rootURL                  : 'open-event-frontend',
```

Comment:
This should not be hardcoded in the repo

Reply:
But the addon needs it to be this way thoughhttps://stackoverflow.com/a/24788051/6870128

Reply:
Two things.

Reply:
1. You don't really need an addon for such a simple task. Just ember's build command and travis's github pages release feature is more than enough.
2. You can modify that just for the build. Not hard coded in the repo. (Hint. you can use an env var to set it)